ansible/playbooks/AdoptOpenJDK_Unix_Playbook/infra_githg.yml

```diff
@@ -0,0 +1,32 @@
+---
+###################################
+# AdoptOpenJDK - Ansible Playbook #
+###################################
+- hosts: "{{ groups['Vendor_groups'] | default('localhost:infra') }}"
+  gather_facts: yes
+  tasks:
+    - block:
+      # Set standard variables
+        - name: Load AdoptOpenJDKs variable file
```

Comment:
whitespace indentation

Reply:
That's copied directly from the main one, but I'll fix in both